tracing-core/src/metadata.rs

```diff
@@ -35,17 +35,6 @@ use core::{
 /// _significantly_ lower than that of creating the actual span. Therefore,
 /// filtering is based on metadata, rather than on the constructed span.
 ///
-/// <div class="example-wrap" style="display:inline-block">
-/// <pre class="ignore" style="white-space:normal;font:inherit;">
-///
-/// **Note**: Although instances of `Metadata` cannot
-/// be compared directly, they provide a method [`callsite`][Metadata::callsite],
-/// returning an opaque [callsite identifier] which uniquely identifies the
-/// callsite where the metadata originated. This can be used to determine if two
-/// `Metadata` correspond to the same callsite.
```

Comment:
i think it would be nice if the `Metadata` type's docs still mentioned the existence of the callsite identifier type somewhere? we should also add a note that `Metadata`s are only equal if they originate from the same callsite, even if all the string values and stuff are the same.